ts-obs/cmd/chunkIntervalSet.go

```diff
@@ -0,0 +1,67 @@
+package cmd
+
+import (
+    "context"
+    "errors"
+	"fmt"
+    "os"
+    "strconv"
+    "time"
+
+	"github.com/spf13/cobra"
+    "github.com/jackc/pgx/v4/pgxpool"
+)
+
+// chunkIntervalSetCmd represents the chunk-interval set command
+var chunkIntervalSetCmd = &cobra.Command{
+	Use:   "set <metric> <duration>",
+	Short: "Sets chunk interval in minutes for a specific metric",
+	RunE:  chunkIntervalSet,
+}
+
+func init() {
+	chunkIntervalCmd.AddCommand(chunkIntervalSetCmd)
+}
+
+func chunkIntervalSet(cmd *cobra.Command, args []string) error {
+    var err error
+
+    if len(args) != 2 {
+        return errors.New("\"ts-obs metrics set-metric-chunk\" requires 2 arguments")
+    }
+
+    if os.Getenv("PGPASSWORD_POSTGRES")  == "" {
```

Comment:
The secret doesn't update if the password is changed.

Reply:
people should update their secrets, because when spawning more pods the secret will be used to populate the environment variables, changing the pass in the db only will break you're whole obs stack